DigitalLearningSolutions.Web/Views/ResetPassword/Index.cshtml

```diff
@@ -1 +1,60 @@
-<h1>Reset Password</h1>
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@model PasswordViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Reset Your Password": "Reset Your Password";
+  var passwordError = ViewData.ModelState["Password"]?.Errors?.Count > 0;
+  var confirmPasswordError = ViewData.ModelState["ConfirmPassword"]?.Errors?.Count > 0;
+  var passwordInputErrorClass = passwordError ? "nhsuk-select--error" : "";
+  var passwordFormErrorClass = passwordError ? "nhsuk-form-group--error" : "";
+  var confirmPasswordInputErrorClass = confirmPasswordError ? "nhsuk-select--error" : "";
+  var confirmPasswordFormErrorClass = confirmPasswordError ? "nhsuk-form-group--error" : "";

```

Comment:
You can use the new TextInputViewComponent to avoid needing all these variables and most of the boilerplate for the inputs themselves.

Reply:
Funky, I'll look into it, thanks.